examples/compare_live_views/README.md

```diff
@@ -0,0 +1,25 @@
+# CompareLiveViews
+
+This is a demo project that illustrates the differences between Popcorn's Local Live View and Phoenix Live View behaviors.
+
+The page is split into two halves, each covered by a modal. One modal is implemented using Local LiveView and the other with Phoenix LiveView.
```

Comment:
It's a bit sad that you can only close the modal :P Let's at least add a button to show the modal again